packages/angular_devkit/build_webpack/BUILD.bazel

```diff
@@ -66,34 +67,32 @@ ts_library(
             "test/**/*",
         ],
     ),
-    deps = [
-        ":build_webpack",
-        "//packages/angular_devkit/architect",
-        "//packages/angular_devkit/architect/node",
-        "//packages/angular_devkit/architect/testing",
+    interop_deps = [
         "//packages/angular_devkit/core",
         "//packages/angular_devkit/core/node",
         "//packages/ngtools/webpack",
-        "@npm//@angular/common",
-        "@npm//@angular/compiler",
-        "@npm//@angular/compiler-cli",
-        "@npm//@angular/core",
-        "@npm//@angular/platform-browser",
-        "@npm//@angular/platform-browser-dynamic",
```

Comment:
do you know why these moved from `ts_project` deps to the `data` deps below? would love to avoid this in general

Reply:
These are used by the generated CLI project used inside the tests.  Treating them as data for the tests is probably more accurate then where they were before since they are not needed for building the actual unit tests.

Reply:
I see, that makes sense then